wondrous-app/components/CreateEntityDiscardTask/index.tsx

```diff
@@ -0,0 +1,29 @@
+import CloseModalIcon from 'components/Icons/closeModal';
```

Comment:
You don't need to write the new component for this, since we already have a modal window with this style. 

If you run `yarn storybook` you can find `Modal` section

![screen-2022-10-05-12-26-55-rrq8n](https://user-images.githubusercontent.com/1454782/194039495-e9d7dafb-d794-493b-8e44-759635d5839c.jpg)


You can replace your code with these: 

```
import Typography from '@mui/material/Typography';
import CloseModalIcon from 'components/Icons/closeModal';
import React from 'react';

import WonderModal from 'components/Modal';
import WonderButton from 'components/Button';

const CreateEntityDiscardTask = ({ open, onClose, onCloseFormModal, entityType }) => {
  const footerRight = (
    <>
      <WonderButton color="grey" onClick={() => onClose(false)}>
        Close
      </WonderButton>
      <WonderButton
        color="primary"
        onClick={() => {
          onCloseFormModal();
          onClose(false);
        }}
      >
        Discard {entityType}
      </WonderButton>
    </>
  );

  return (
    <WonderModal
      title={`Discard ${entityType} data?`}
      onClose={() => onClose(false)}
      footerRight={footerRight}
      maxWidth={560}
      open={open}
    >
      <Typography sx={{ color: 'white' }}>Are you sure you want to discard your current {entityType}?</Typography>
    </WonderModal>
  );
};

export default CreateEntityDiscardTask;

```

and it will look like on the image below: 

![screen-2022-10-05-12-28-03-olzla](https://user-images.githubusercontent.com/1454782/194039709-f99c2cb8-4870-42ab-8d1e-8c2adedcc4a1.jpg)